nss/aad_auth/error_c.go

```diff
@@ -43,6 +43,8 @@ func errToCStatus(ctx context.Context, err error) (nssStatus, errno int) {
 
 	if err != nil {
 		logger.Debug(ctx, "Returning to NSS error: %d with errno: %d", nssStatus, errno)
+	} else {
+		logger.Debug(ctx, "Returning NSS STATUS SUCCESS (%d) with errno: %d", nssStatus, errno)
```

Comment:
No need to add the constant and the integer value, just say "Returning NSS_STATUS_SUCCESS" without "%d" and the errno.